guides/source/action_text_overview.md

```diff
@@ -129,15 +129,15 @@ name the attribute to be something different from `content`.
 Once you have added the `has_rich_text` class method to the model, you can then
 update your views to make use of the rich text editor (Trix) for that field. To
 do so, use a
-[`rich_textarea`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-rich_textarea)
```

Comment:
This link works for me if you replace `api` with `edgeapi`:

https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-rich_textarea

`rich_textarea` and `rich_text_area` will both be supported as aliases for one another in 8.0.